### PR TITLE
merge gobgp-update into master

### PR DESCRIPTION
**Pull request by murali-reddy**

### CODE REVIEW

**Review by aauren — APPROVED** [review]
We've been testing these changes for more than a week now. We've been testing a build of this branch in one of our test clusters for 4+ hours now with no problems.

I recommend that we merge this whenever you feel comfortable @murali-reddy and work out any other issues that pop up after cutting the 1.1-rc1 release.

File: Makefile [hunk 1]
```diff
@@ -239,14 +239,13 @@ endif
 gobgp:
 ifeq "$(BUILD_IN_DOCKER)" "true"
 	@echo Building gobgp
-	$(DOCKER) run -v $(PWD)/vendor:/go/src -w /go/src/github.com/osrg/gobgp/gobgp $(DOCKER_BUILD_IMAGE) \
-    sh -c 'GOARCH=$(GOARCH) CGO_ENABLED=0 go build -o gobgp'
+	$(DOCKER) run -v $(PWD):/go/src/github.com/cloudnativelabs/kube-router -w /go/src/github.com/cloudnativelabs/kube-router $(DOCKER_BUILD_IMAGE) \
+    sh -c 'go get -u github.com/osrg/gobgp && GOARCH=$(GOARCH) CGO_ENABLED=0 go build -o gobgp github.com/osrg/gobgp/cmd/gobgp'
```
Comment (mrueg) [inline Makefile:243]: do you want to version lock the go-get call here? 
```suggestion
    sh -c 'go get -u github.com/osrg/gobgp@v2.19.0 && GOARCH=$(GOARCH) CGO_ENABLED=0 go build -o gobgp github.com/osrg/gobgp/cmd/gobgp'
```

File: Makefile [hunk 2]
```diff
@@ -239,14 +239,13 @@ endif
...
 	@echo Finished building gobgp.
 else
-	cd vendor/github.com/osrg/gobgp/gobgp && \
-	CGO_ENABLED=0 GOARCH=$(GOARCH) GOOS=linux go build -o gobgp
+	go get -u github.com/osrg/gobgp && \
```
Comment (mrueg) [inline Makefile:246]:
```suggestion
	go get -u github.com/osrg/gobgp@v2.19.0 && \
```

Comment (murali-reddy) [reply Makefile:246]: go get refers to go.mod and attempts to get the required version.

File: Makefile (hunk 1 above)
Comment (murali-reddy) [reply Makefile]: go get refers to go.mod and attempts to get the required version.

**Comment by murali-reddy** [comment]
Thanks @aauren for testing again. @mrueg thanks for sanity checking. Addressed your review comments.

**Pull request merged**